openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -43,12 +42,16 @@
     CertificateStatuses.downloadable,
 ]
 
-# Maximum number of retries before giving up. For reference, 11 retries with exponential backoff yields a maximum
-# waiting time of 2047 seconds (about 30 minutes). Setting this to None could yield unwanted behavior: infinite retries.
-MAX_RETRIES = 11
```

Comment:
[nonblocking] is there a particular reason why you took this out of being the constant?

Reply:
I just figured we didn't need it anymore, with the number of retries being part of the task declaration. It was only used in the retry logic for this task too, and I didn't think it needed to be a constant anymore.